### PR TITLE
Night theme for UI

### PR DESCRIPTION
**Pull request by DmitrySvetlichny**

### CODE REVIEW

File: Sources/Controllers/RouteInfoMenu/OARouteInfoView.mm [hunk 1]
```diff
@@ -241,9 +241,9 @@ - (void) applyCornerRadius:(BOOL)enable
 - (void) setupGoButton
 {
     BOOL isActive = _app.data.pointToNavigate != nil;
-    _goButton.backgroundColor = isActive ? UIColorFromRGB(color_primary_purple) : UIColorFromRGB(color_button_gray_background);
-    [_goButton setTintColor:isActive ? UIColor.whiteColor : UIColorFromRGB(color_text_footer)];
-    [_goButton setTitleColor:isActive ? UIColor.whiteColor : UIColorFromRGB(color_text_footer) forState:UIControlStateNormal];
+    _goButton.backgroundColor = isActive ? UIColor.buttonBgColorPrimary : UIColor.buttonBgColorSecondary;
+    [_goButton setTintColor:isActive ? UIColor.buttonTextColorPrimary : UIColor.buttonTextColorSecondary];
```
Comment (tigrim) [inline Sources/Controllers/RouteInfoMenu/OARouteInfoView.mm:245]: UIColor *color = isActive ? UIColor.buttonTextColorPrimary : UIColor.buttonTextColorSecondary;

File: Sources/Views/Banners/FreeFavoritesBackupBanner/FreeBackupBanner.swift
Comment (tigrim) [inline Sources/Views/Banners/FreeFavoritesBackupBanner/FreeBackupBanner.swift:1]: FreeBackupBanner.xib - were any edits made here?

File: Sources/Controllers/DashboardOnMap/MapSettings/OAMapSettingsNauticalDepthScreen.m [hunk 2]
```diff
@@ -125,7 +125,7 @@ - (UITableViewCell*)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NS
 
             BOOL isOn = [_depthContours.value isEqualToString:@"true"];
             cell.leftIconView.image = [UIImage templateImageNamed:item.iconName];
-            cell.leftIconView.tintColor = isOn ? UIColorFromRGB(item.iconTint) : UIColorFromRGB([item integerForKey:@"iconTintDisabled"]);
+            cell.leftIconView.tintColor = isOn ? item.iconTintColor : [item objForKey:@"iconTintDisabled"];
```
[the remark this answers is not in the document]
Comment (DmitrySvetlichny) [reply Sources/Controllers/DashboardOnMap/MapSettings/OAMapSettingsNauticalDepthScreen.m:128]: Yes

File: Sources/Views/Banners/FreeFavoritesBackupBanner/FreeBackupBanner.swift
Comment (DmitrySvetlichny) [reply Sources/Views/Banners/FreeFavoritesBackupBanner/FreeBackupBanner.swift:1]: Yes

**Review by tigrim — APPROVED** [review]

**Pull request merged**